bridge/server/server.ts

```diff
@@ -16,6 +16,8 @@ try {
 // init destination and debug flags
 logger.configure(configuration.logging.destination, configuration.logging.enabledComponents);
```

Comment:
I think you need to add the `defaultLogLevel` here to make it work as expected. Without it, I couldn't get any debug logs even after setting the log level to `debug`
```suggestion
logger.configure(
  configuration.logging.destination,
  configuration.logging.enabledComponents,
  configuration.logging.defaultLogLevel
);
```